app/models/resource.rb

```diff
@@ -38,23 +38,24 @@ class Resource < ActiveRecord::Base
       # specify the list of attributes available for faceting
       attributesForFaceting %i[categories open_times]
       # Define attributes used to build an Algolia record
-      add_attribute :_geoloc do
-        { lat: address_latitude.to_f, lng: address_longitude.to_f }
```

Comment:
Is removing this going to prevent Algolia from properly ordering by distance?

Reply:
Ah, [you are correct](https://www.algolia.com/doc/guides/managing-results/refine-results/geolocation/). Adding back.

Reply:
This line is causing the TestCafe test for adding a new resource to fail, but I can't figure out how this used to work before. The error message is:

```rb
 NoMethodError (undefined method `latitude' for nil:NilClass)
```

However, I don't see where `address_latitude` or `address_longitude` were defined, and therefore how this handled the case where there is no latitude or longitude. Do you know where they were defined?